src/api/app/controllers/source_controller.rb

```diff
@@ -516,12 +516,12 @@ def update_project_meta
     render_ok
   end
 
-  def check_and_remove_repositories!(repositories, full_remove, force = false)
+  def check_and_remove_repositories!(repositories, full_remove, force = false, skip_backend = false)
```

Comment:
This method is only ever used in this file, always with `skip_backend=false`. Why do you make skip_backend a parameter? :-)